src/allmydata/scripts/tahoe_ls.py

```diff
@@ -64,13 +64,14 @@ def list(options):
         print(quote_output(data, quotemarks=False), file=stderr)
         return 1
 
+    path = unicode(path, "utf-8")
```

Comment:
There is one use of `path` earlier in this function.  Is it okay or are is it just not exercised by any of the tests (or at least the tests being ported here)?